collection-scripts/gather_monitoring

```diff
@@ -104,6 +107,28 @@ alertmanager_get() {
       2> "$result_path.stderr"
 }
 
+# metrics_get runs "promtool tsdb dump" inside a prometheus instance and retrieves the samples.
+# TODO: maybe move it to a new gather_metrics?
+metrics_get() {
+  local result_path="${MONITORING_PATH}/prometheus/metrics"
+  mkdir -p "$(dirname "${result_path}")" 
+
+  # TODO: make sure it's a healthy prometheus (in case one of them is dowm)? get metrics from both?
+  prometheus_pod=${PROM_PODS[0]}
+  # TODO: allow customizing. (override default or extend?)
```

Comment:
we'll need to allow users to customize the following flags: https://prometheus.io/docs/prometheus/latest/command-line/promtool/#promtool-tsdb-dump-openmetrics

I don't see any util for args parsing, should I add one? (it'll be ugly) or should we just "trust" the user  and blindly pass their "$@"? Doing so we'll no be able to have default matchers (metrics that we want to always fetch).